client/helpdlg.cpp

```diff
@@ -886,6 +886,12 @@ void help_widget::set_topic_unit(const help_item *topic, const char *title)
                       utype_build_shield_cost_base(max_utype));
     add_info_progress(_("Firepower:"), utype->firepower, 0,
                       max_utype->firepower);
+    add_info_progress(
+        _("Vision:"),
+        static_cast<int>(sqrt(static_cast<double>(utype->vision_radius_sq))),
+        0,
+        static_cast<int>(
+            sqrt(static_cast<double>(max_utype->vision_radius_sq))));
```

Comment:
Doing `std::sqrt(x)` (note the `std::` forcing usage of the C++ function, as opposed to C) is equivalent to `sqrt(static_cast<double>(x))`. If precision isn't an issue, `sqrt(x)` will also work just fine (might just cast to `float`, not sure what the rules are).

Reply:
Should I change this to `std::sqrt()` without the inner cast then?

Reply:
Would be more readable